docs/platform/release-notes/tizen-5-5-m1.md

```diff
@@ -0,0 +1,630 @@
+
+
+## System (IoT System)
+
+### New and changed features
+
+- Memory management for long-lived process like background-locked application has been enhanced.
+  - Separated memory cgroup for long-lived process has been added. When system goes to idle state, it proactively reclaims the long-lived memory of the process to provide better memory usage environment for devices with limited memory.
+- Memory reclaim at boot can be configured (enabled and disabled).
+- Watchdog functionalities for headless-device have been enhanced like headed-device.
+- System-dump supports extension mechanism to dump vendor-specific information. In addition, system-dump dumps installed-package list for checking installed-app version as default behaviors.
+
+
+## System (Base)
+
+### New and changed features
+
+- Open source upgrade
+  - icu (63.1)
+  - python (2.7.15)
+  - sqlite (3.26.0)
+  - tzdata (2018f)
+  - gobject-introspection (1.52.1)
+  - python-gobject (3.30.1)
+  - python-cairo (1.17.0)
+  - expat (2.2.6)
+  - openfst (1.6.5) has been added.
+  - file (5.35)
+  - jsoncpp (1.8.4)
+  - jsontool (9.0.6)
+  - psmisc (23.2)
+  - re2 (20181001)
+  - gtk-doc (1.29)
+  - xz (5.2.4)
+  - itstool (2.0.5) has been added.
+  - meson (0.49)
+  - python-setuptools (40.6.3)
+  - dos2unix (7.4.0)
+  - libarchive (3.3.3)
+  - libsolv (0.6.35)
+  - perl-TimeDate (2.30)
+  - re2c (1.1.1)
+- Expanded the capi-system-settings API set:
+  - Added system-settings key value
+    - SYSTEM_SETTINGS_KEY_ACCESSIBILITY_GRAYSCALE
+    - SYSTEM_SETTINGS_KEY_ACCESSIBILITY_NEGATIVE_COLOR
+  - Added features for accessibility
+    - http://tizen.org/feature/accessibility.grayscale
+    - http://tizen.org/feature/accessibility.negative
+
+### Fixes
+
+- CVE patches
+  - libxslt : CVE-2019-11068
+  - sqlite3: CVE-2019-9936
+  - python: CVE-2019-9740 and CVE-2019-9636
+  - file : CVE-2019-8906
+
+
+## Application framework
+
+### New and changed features
+
+- New notification API set
+  - Various types of notifications have been provided to represent various information.
+- Improved app-control API set
+  - API set for Uniform Resource Identifier (URI) has been provided.
+  - A synchronous launch request API has been provided.
+  - It allows applications to use API set that can be used separately for each request of app-control.
+- Improved API set for package manager
+  - A way to describe dependencies between packages has been provided.
+  - API to get dependencies between packages has been provided.
+- New C# API set
+  - C# API set for watch-complication has been provided.
+  - C# API set for application events has been provided.
+
+
+## Window System
+
+### New and changed features
+
+- Wayland
+  - The open source wayland-protocols has been upgraded to version 1.16.
+- Wayland extension
+  - The tizen_launch_appinfo interface has been added. Display server can get the application information such as pids and appids from the clients.
```

Comment:
As per the comment in PR 707, to maintain consistency with line number 335: Support fo getting the PID of ....

Please change pids to PIDs:

..... information such as PIDs and appids from the clients.

Also, please verif, if we can change appids to APPIDs?

Reply:
We will use 'pid' and 'appid' instead of 'PID' and 'APPID' for consistency.
Two 'PID's will be changed into 'pid'